src/ui.text-alert/configs/default.config.js

```diff
@@ -32,6 +37,24 @@ export default /*tw*/ {
       },
     },
   },
+  title: {
+    base: "font-bold text-lg leading-tight mb-1",
+  },
+  description: {
+    base: "text-sm leading-normal mb-2",
+  },
+  leftSlot: {
+    base: "flex justify-start items-center gap-5",
+  },
+  rightSlot: {
+    base: "flex justify-end items-center gap-5",
+  },
+  topSlot: {
+    base: "flex justify-center items-start gap-5",
+  },
+  bottomSlot: {
```

Comment:
styling for slots and relative to them divs no needed, user will style it by himself whatever they need.